dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -402,8 +404,10 @@
     <Style x:Key="NavigationViewOverflowButtonStyleWhenPaneOnTop" TargetType="Button">
         <Setter Property="Background" Value="{ThemeResource NavigationViewItemBackground}" />
         <Setter Property="Foreground" Value="{ThemeResource TopNavigationViewItemForeground}" />
-        <Setter Property="Height" Value="{StaticResource PaneToggleButtonHeight}" />
-        <Setter Property="Width" Value="{StaticResource PaneToggleButtonWidth}" />
```

Comment:
@StephenLPeters @YuliKl Please take a look at this and say if you are fine with this change or not.

The width of the overflow button was set to 40px by the PaneToggleButtonWidth resource. As we decided to increase its width to 48px, changing that resource value would also lead to an increased width for the PaneToggle button which also consumes that resource.

As such, I added a new dedicated resource here to control the width of the overflow button so that the PaneToggle button width will remain unchanged. While this will break customization for developers who used that resource to style the width of the overflow button, I think this is such a minor case here that we should be fine and that the benefits outweigh the possible small inconvenience for developers (if any are caused at all). I also don't know how we would be able to increase the width of that overflow button while keeping the width of the PaneToggle button untouched and not causing a potential customization break....

Once I introduced a dedicated TopNavigationViewOverflowButtonWidth I also created its matching counterpart for the height - TopNavigationViewOverflowButtonHeight.